ember-cli-build.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-env node */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
+  const app = new EmberAddon(defaults, {
```

Comment:
If we use our current set of lint rules, there will be differences between the auto-generated `ember-cli` scaffold and our apps/addons anyway. Either we're gonna have to see additional `// eslint-disable no-var` statements, or we're gonna see `let` and `const` where `ember-cli` spit out `var`. I would argue the latter is less mentally taxing than the former.

As for updating all other addons to conform to this one, as soon as they update to use the new lint rules, they will also have to make this change.